Xamarin.Forms.Platform.Android/Forms.cs

```diff
@@ -70,7 +70,7 @@ public static Color GetColorButtonNormal(Context context)
 		// Why is bundle a param if never used?
 		public static void Init(Context activity, Bundle bundle)
 		{
-			Assembly resourceAssembly = Assembly.GetCallingAssembly();
+			Assembly resourceAssembly = Assembly.GetEntryAssembly ();
```

Comment:
Does this work? I tested this yesterday briefly when trying to find a better approach and in my tests this returns null

![image](https://user-images.githubusercontent.com/5375137/53032961-2e82af80-342d-11e9-8741-7dc60508eed9.png)

Reply:
https://bugzilla.xamarin.com/show_bug.cgi?id=21097

```
Assembly.GetEntryAssembly() is supposed to return the _executable_ (.exe file) that was used to launch the app. Xamarin.Android apps don't contain .exe files; they consist of .dll files.

http://msdn.microsoft.com/en-us/library/system.reflection.assembly.getentryassembly(v=vs.110).aspx
> Gets the process executable in the default application domain.

Consequently, Xamarin.Android apps don't have an entry assembly, and Assembly.GetEntryAssembly() returns `null`.
```

Reply:
👍 

That's kinda surprising. I assumed whichever assembly contained the `Main` method would've been returned :)